### ISSUE TITLE
[VoiceOver] Reader Post Detail: Navigation is inconsistent

### ISSUE THREAD

**Issue by shiki** [wordpress-mobile/WordPress-iOS#12904]
## Expected behavior

I think navigating (swiping right) should be left to right, top to bottom.

## Actual behavior

As of now, navigating with VoiceOver can go like this:

1. Site name + URL
2. More Button
3. Post Title
4. Author + Date
5. Save button ❌ (not the first paragraph)

Or like this:

1. View controller title 
2. Share button
3. The content's first paragraph ❌ (not the Site name) 

Please see [this video](https://drive.google.com/file/d/1-t3npWYakWyO9PeIvC_1c-LQkWNuEhAY/view?usp=sharing). 

### Steps to reproduce the behavior

1. Turn VoiceOver on
2. Navigate to Reader and to a post
3. Navigate by swiping right (or left to go back)

Tested on iPhone XS, iOS 13.1.3, WPiOS `develop` (00ff736976e19e006c6762ac3f53165afcf556f7)

**Comment by shiki** [comment]
I tried to look into this but I'd need some help on understanding how the Reader Detail's UI works. An expert on Reader could probably fix this pretty quickly. 🙂

**Comment by rezzap** [comment]
There was some concerned raised about this in an app review:

Translation:

> It would also be good if article titles were displayed as headings in the reader, as it used to be. Until recently, when using a Bluetooth keyboard, after reading an article in the reader, it was still possible to return to the article overview in the reader with the escape key, which is no longer possible. Especially with long articles it is tedious to navigate back up if the article title is no longer marked as a heading and the back function with esc on the keyboard is no longer available.

Original:

> Weiters wäre es gut, wenn Beitragstitel im Reader als Überschriften dargestellt werden würden, so wie es früher war. Bis vor kurzem war es außerdem noch möglich, bei nutzen einer Bluetooth Tastatur, nach dem Lesen eines Beitrags im Reader, mit der Taste escape zur Beitragsübersicht im Reader zurückzukehren, das geht jetzt auch nicht mehr. Gerade bei langen Beiträgen ist es mühsam, wieder nach oben zu navigieren, wenn der Beitragstitel nicht mehr als Überschrift gekennzeichnet ist, und die Zurück-Funktion mit esc auf der Tastatur nicht mehr zur Verfügung stehen.

Reference iOS review by hebejacq – Edited Nov 26, 2020
Version 16.1.1
